<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="21" started-at="2014-08-14T06:19:52Z" finished-at="2014-08-14T06:19:52Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="21" started-at="2014-08-14T06:19:52Z" finished-at="2014-08-14T06:19:52Z">
      <class name="com.apalya.testcases.sample_play_check">
        <test-method status="FAIL" signature="sample_play(java.lang.String)[pri:0, instance:com.apalya.testcases.sample_play_check@876e4b8]" name="sample_play" duration-ms="0" started-at="2014-08-14T11:49:52Z" data-provider="registerData" finished-at="2014-08-14T11:49:52Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Method public void com.apalya.testcases.sample_play_check.sample_play(java.lang.String) requires a @DataProvider named : registerData]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Method public void com.apalya.testcases.sample_play_check.sample_play(java.lang.String) requires a @DataProvider named : registerData
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:268)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:417)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sample_play -->
      </class> <!-- com.apalya.testcases.sample_play_check -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
